src/ImageSharp/Formats/Gif/GifEncoderCore.cs

```diff
@@ -74,7 +79,13 @@ public void Encode<TPixel>(Image<TPixel> image, Stream stream)
             this.bitDepth = ImageMaths.GetBitsNeededForColorDepth(quality);
 
             // Quantize the image returning a palette.
-            QuantizedImage<TPixel> quantized = ((IQuantizer<TPixel>)this.Quantizer).Quantize(image, quality);
+            this.hasMultipleFrames = image.Frames.Any();
```

Comment:
I know this is correct but the statement looks confusing when you look at it the first time. Maybe you should call it `hasFrames` instead?